hack/configure_local_webhook.sh

```diff
@@ -89,3 +89,11 @@ webhooks:
 EOF_CAT
 
 oc apply -n openstack -f ${TMPDIR}/patch_webhook_configurations.yaml
+
+# Scale-down operator deployment replicas to zero and remove OLM webhooks
+CSV_NAME="$(oc get csv -n openstack-operators -l operators.coreos.com/neutron-operator.openstack-operators -o name)"
+
```

Comment:
Should be good to have a backup of csv so can be reused when re deploying with OLM as mentioned in dev docs https://github.com/openstack-k8s-operators/dev-docs/blob/main/running_local_operator.md#disabling-a-service-within-the-csv

Reply:
Could we consider this in a follow-up?

Reply:
should be good to have it in this patch itself as we changing how this work currently

Reply:
This patch is only automating what the user was already having to do manually before running `make run-with-webhook` [1].  So in that sense, this patch is not changing anything.

[1] https://github.com/openstack-k8s-operators/dev-docs/blob/main/webhooks.md#using-webhooks-locally

Reply:
hmm we following different links, i was specifically pointing to other one https://github.com/openstack-k8s-operators/dev-docs/blob/main/running_local_operator.md#disabling-a-service-within-the-csv that refers to https://github.com/openstack-k8s-operators/dev-docs/blob/main/webhooks.md#disabling-webhooks

Reply:
Right.  Either way, the user had to already manually drop OLM replicas for the controller-manager to 0 and remove the OLM webhooks.  They can still do that manual process first, and then when this script runs it will effectively be a no-op in terms of CSV changes.  But anyhow, I will add an auto-backup here to set a precedent that can be echoed in this same logic across all our operators.

Reply:
@karelyatin Do you think we should always back-up the CSV whenever the script is run, or only if the CSV has `deployments > 0` and still has the `webhookdefinitions`?  i.e. if they keep running `make run-with-webhook` and `ctrl+C` and repeat, then every iteration after the first would be backing up the already-modified CSV.

Reply:
@karelyatin Pushed a functioning prototype that seems to work.  Please let me know what you think.